group.py

```diff
@@ -2,4 +2,13 @@
 
 # Your code to go here...
 
-my_group =
+
+Jill={"name":"Jill","age":26,"Job":"Biologist","Relation":["friend","partner"]}
+Zalika={"name":"Zalika","age":28,"Job":"artist","Relation":["friend","renter"]}
+John={"name":"John","age":27,"Job":"writer","Relation":["partner","uncle"]}
+Nash={"name":"Nash","age":34,"Job":"chef","Relation":["cousin","landlord"]}
+
```

Comment:
I have to say this method (dictionary) is not pretty convenient when creating new person, so yeah, I agree that class is more suitable to it.